go/vt/topo/etcd2topo/server.go

```diff
@@ -34,13 +34,21 @@ We follow these conventions within this package:
 package etcd2topo
 
 import (
+	"flag"
 	"strings"
 	"time"
 
 	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/pkg/transport"
 	"vitess.io/vitess/go/vt/topo"
 )
 
+var (
+	clientCertPath = flag.String("topo_etcd2_tls_cert", "", "the client cert to use to connect to the etcd topo server, requires topo_etcd2_tls_key, enables TLS")
+	clientKeyPath  = flag.String("topo_etcd2_tls_key", "", "the client key to use to connect to the etcd topo server, enables TLS")
+	clientCaPath   = flag.String("topo_etcd2_tls_ca", "", "the ca to use to validate clients when connecting to the etcd topo server")
```

Comment:
This is for validating the server, not clients.

```suggestion
	serverCaPath   = flag.String("topo_etcd_tls_ca", "", "path to the ca to use to validate the server cert when connecting to the etcd topo server")
```